File: app/src/main/java/com/absinthe/anywhere_/adapter/page/PageTitleNode.java
package com.absinthe.anywhere_.adapter.page;

import com.chad.library.adapter.base.entity.node.BaseExpandNode;
import com.chad.library.adapter.base.entity.node.BaseNode;

import org.jetbrains.annotations.Nullable;

import java.util.List;

public class PageTitleNode extends BaseExpandNode {

    private List<BaseNode> childNode;
    private String title;

    public PageTitleNode(List<BaseNode> childNode, String title) {
        this.childNode = childNode;
        this.title = title;

        setExpanded(false);
    }

    public String getTitle() {
        return title;
    }

    @Nullable
    @Override
    public List<BaseNode> getChildNode() {
        return childNode;
    }
}

File: app/src/main/java/com/absinthe/anywhere_/adapter/page/PageTitleNode.kt
package com.absinthe.anywhere_.adapter.page

import com.chad.library.adapter.base.entity.node.BaseExpandNode
import com.chad.library.adapter.base.entity.node.BaseNode

class PageTitleNode(override val childNode: MutableList<BaseNode>, val title: String) : BaseExpandNode() {

    init {
        isExpanded = false
    }

}